content/javascript/concepts/arrays/terms/findLastIndex/findLastIndex.md

```diff
@@ -0,0 +1,84 @@
+---
+Title: '.findLastIndex()'
+Description: 'Iterates through the array in reverse order and returns the index that passes the provided testing function. If no item passes, it returns -1.'
+Subjects:
+  - 'Web Development'
+  - 'Computer Science'
+Tags:
+  - 'Arrays'
+  - 'Methods'
+CatalogContent:
+  - 'introduction-to-javascript'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The `.findLastIndex()` iterates through the array in reverse order and returns the index that passes the provided testing function. If no item passes, it returns -1.
+
+## Syntax
+
+```pseudo
+array.findLastIndex((element, index) => {...});
+```
+
+The function can be invoked with two arguments:
+
+- `element`: The current element we are iterating through.
+- `index` (optional): The index of the array element.
+
+## Example
```

Comment:
I would recommend deleting this example. This is essentially a duplicate from the findLast entry and it isn't really demonstrating anything that isn't present in the second example .